<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchFrame</class>
 <widget class="QFrame" name="SearchFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="btn_rewrite">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>重写</string>
   </property>
  </widget>
  <widget class="QTableView" name="tb_result">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>110</y>
     <width>430</width>
     <height>300</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>检索词类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tv_title">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>52</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>检   索   词</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serach_type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
